Common/TableProducer/PID/pidTPCFull.cxx

```diff
@@ -136,6 +148,36 @@ struct tpcPidFull {
       LOGP(info, "Loading TPC response from CCDB, using path: {} for ccdbTimestamp {}", path, time);
       response.PrintAll();
     }
+
+    if (useNetworkCorrection) {
```

Comment:
Maybe you could consider using a helper function in the `pidTPCML.h` so that we don't duplicate the code?